### PR TITLE
add aten::floor_divide

### PR DESCRIPTION
**Pull request by bszmelcz**
## Details:
- [x] rename files
- [x] update implementation 
- [x] add layer test for `aten::floor_divide` operator
## Ticket:
99224

### CODE REVIEW

File: tests/layer_tests/pytorch_tests/test_floor_divide.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# Copyright (C) 2018-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import pytest
+import numpy as np
+from pytorch_layer_test_class import PytorchLayerTest
+
+
+class Testfloor_divide(PytorchLayerTest):
```
Comment (sikorsl1) [inline tests/layer_tests/pytorch_tests/test_floor_divide.py]: I think that following naming convention it should be `TestFloorDivide`

File: tests/layer_tests/pytorch_tests/test_floor_divide.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    def _prepare_input(self):
+        return (self.input_tensor, self.other_tensor)
+
+    def create_model(self):
+
+        import torch
+
+        class aten_floor_divide(torch.nn.Module):
+            def __init__(self):
+                super(aten_floor_divide, self).__init__()
```
Comment (sikorsl1) [inline tests/layer_tests/pytorch_tests/test_floor_divide.py:19]: Why don't you simply use `super().__init__()` ? Probably it is not a problem, but I am curious ;)

Comment (bszmelcz) [reply tests/layer_tests/pytorch_tests/test_floor_divide.py:19]: Just a convention that I am used to :)

File: tests/layer_tests/pytorch_tests/test_floor_divide.py (hunk 1 above)
Comment (bszmelcz) [reply tests/layer_tests/pytorch_tests/test_floor_divide.py]: Yeah, I am gonna fix it

File: src/frontends/pytorch/src/op_table.cpp [hunk 3]
```diff
@@ -121,7 +121,7 @@ const std::map<std::string, CreatorFunction> get_supported_ops() {
         {"aten::expand", op::translate_expand},
         {"aten::expand_as", op::translate_expand_as},
         {"aten::flatten", op::translate_flatten},
-        {"aten::floordiv", op::translate_floordiv},
```
Comment (mvafin) [inline src/frontends/pytorch/src/op_table.cpp:124]: Why you delete `floordiv`?

Comment (bszmelcz) [reply src/frontends/pytorch/src/op_table.cpp:124]: @mvafin Oh I thought it is the same operator. I have reverted the change and add seperate file for `aten::floor_divide`

**Review by mvafin — APPROVED** [review]

**Review by sikorsl1 — APPROVED** [review]

**Pull request merged**